components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,6 +63,31 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        if (state.entities.cloud.products && subscription) {
+            const product = state.entities.cloud.products[subscription?.product_id];
+            return product.product_family === 'cloud' ? product : null;
+        }
+        return null;
+    });
+
+    let subscriptionPlan = 'CLOUD_PROFESSIONAL';
+
+    switch (product?.name) {
+    case 'Mattermost Cloud Starter':
+        subscriptionPlan = 'CLOUD_STARTER';
+        break;
+    case 'Mattermost Cloud Professional':
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    case 'Mattermost Cloud Enterprise':
+        subscriptionPlan = 'CLOUD_ENTERPRISE';
+        break;
```

Comment:
The product names probably won't include the word "Mattermost" in each, but regardless, should we maybe rely on the nickname that we can keep the same instead of the actual name? @nickmisasi thoughts ?

Reply:
@nickmisasi have you already defined how this nickname will look like?

Reply:
Products don't have nicknames. Only prices do. With the multiple products model, this is the only real way we can do this, unless you want to add more metadata

Reply:
How hard would it be to add a nickname in the products model @nickmisasi?  @marianunez I think we could use some nickname metadata, something that looks more like an actual key for identifying the tier and not relying in the names, which is a very error prone approach. Any thoughts on this?

Reply:
Will this logic in the CWS also use names for the products? At the end is the same concept but in the backend. Not sure how much we gain there

Reply:
If we do go with the name approach, we may need to only check if it contains `Starter` or `Professional` but not sure that the actual name will be that verbose like `Mattermost Cloud Starter`

Reply:
ok, I will go with the checking if the name contains a reference for starter, professional or enterprise

Reply:
See here: https://github.com/mattermost/mattermost-webapp/pull/8094#discussion_r632726404

Reply:
Here's what the products will look like with this latest SKU change:

```
[
    {
        "id": "prod_JLoXfwr9wf0vZn",
        "name": "Mattermost Cloud Standard",
        "description": "This product is used to help testing for multiple cloud tiers",
        "price_per_seat": 4,
        "add_ons": null,
        "sku": "cloud-starter",
        "price_id": "price_1IpfPOC2mR71ZtK2jyMhVIjx",
        "product_family": "cloud"
    },
    {
        "id": "prod_J62eIfR2GZBQU0",
        "name": "Mattermost Cloud Professional",
        "description": "",
        "price_per_seat": 10,
        "add_ons": null,
        "sku": "cloud-professional",
        "price_id": "price_1ITqZPC2mR71ZtK2BP1KVRgs",
        "product_family": "cloud"
    }
]
```

Reply:
got it!

Reply:
done @nickmisasi